app/assets/stylesheets/modules/_markdown_it.sass

```
@@ -0,0 +1,37 @@
+.js-markdown-view
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6
+    &::before
+      display: block
+      content: " "
+      margin-top: -3.75rem
+      height: 3.75rem
+      visibility: hidden
+      pointer-events: none
```

@machida 
お疲れさまです。
申し訳ありませんが、私の書いた`before` のcssが他の部分を覆い隠してしまうようです。ですので、テストが何回もパスできません。
代わりに `scroll-margin-top`を使用できますか？ローカルでやりました、そしてそれはうまくいきます。
https://developer.mozilla.org/ja/docs/Web/CSS/scroll-margin-top
```suggestion
    scroll-margin-top: 3.75rem
```